Sources/App/Configuration/Environment.swift

```diff
@@ -1,57 +1,78 @@
 import Mailgun
 import Vapor
 
-// MARK: App User Access Token
+
 extension Environment {
-    static var appUserAccessTokenSignerKey: Data {
-        assertGet("APP_USER_ACCESS_TOKEN_SIGNER_KEY").data(using: .utf8)!
-    }
+    struct AppUser {
 
-    static var appUserAccessTokenExpiration: TimeInterval {
-        self.get("APP_USER_ACCESS_TOKEN_EXPIRATION").flatMap { TimeInterval($0) } ?? 3600
-    }
-}
+        // MARK: App User Access Token
 
-// MARK: App User Refresh Token
-extension Environment {
-    static var appUserRefreshTokenSignerKey: Data {
-        assertGet("APP_USER_REFRESH_TOKEN_SIGNER_KEY").data(using: .utf8)!
-    }
+        struct AccessToken {
+            static var signerKey: Data {
+                assertGet("APP_USER_ACCESS_TOKEN_SIGNER_KEY").data(using: .utf8)!
+            }
 
-    static var appUserRefreshTokenExpiration: TimeInterval {
-        self.get("APP_USER_REFRESH_TOKEN_EXPIRATION").flatMap { TimeInterval($0) } ?? 3600 * 24 * 365
-    }
-}
+            static var expiration: TimeInterval {
+                TimeInterval(assertGet("APP_USER_ACCESS_TOKEN_EXPIRATION")) ?? 3600
+            }
+        }
```

Comment:
There seems to issue with indentation.

Reply:
the indentations here are correct, the env vars have been re-organized a bit so they are now indented one level deeper than before